source/standards/logging.html.md.erb

```diff
@@ -145,23 +175,30 @@ Example:
 }
 ```
 
-### Advice for particular frameworks or platforms
+## Advice for particular frameworks or platforms
+
+### Dropwizard
 
-#### Dropwizard
+The [dropwizard-logstash] library will set things up for you using the
+standard names.
```

Comment:
Are we sure this advice is applicable to Splunk. As Logstash format is specific to Kibana (used by Logit)

Reply:
Good point. I _think_ Splunk would handle the Logstash format fine, so isn't _incorrect_. There is some Splunk guidance for Java logging but I'm not sure I'd want to recommend proprietary libraries: https://dev.splunk.com/enterprise/docs/java/logging-java